test/javascripts/acceptance/mobile-chat-test.js

```diff
@@ -34,4 +37,22 @@ acceptance("Discourse Chat - Mobile test", function (needs) {
     await click(".chat-channel-row.chat-channel-7");
     assert.notOk(exists(".chat-full-screen-button"));
   });
+
+  test("Chat new personal chat buttons", async function (assert) {
+    await visit("/latest");
```

Comment:
Maybe we can just use a `visit("chat/channel...")` call instead of these actions